backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/WorkAtOrganization.kt

```diff
@@ -13,7 +13,8 @@ data class WorkAtOrganization(
     val responsibility: ShortTextInput,
     val workSinceDate: DateInput,
     val payment: Boolean,
-    val certificate: Attachment?
+    val certificate: Attachment?,
+    val isAlreadyVerified: Boolean?
```

Comment:
```suggestion
    val isAlreadyVerified: Boolean = false
```
:wrench: I think you can just set the default value here to avoid the check below

Reply:
The check below will still be needed, as the prop is optional. It has to stay optional otherwise it will be an error when calling without this prop. Setting a default does not make it optional in the graphql scheme.

Reply:
ah alright i see the issue. thx for the explanation. Then its fine